deploy/kessel-inventory.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: inventory
+objects:
+  - apiVersion: v1
+    kind: Secret
```

Comment:
are we supposed to add this to the template? 
AFAIK app-interface will create the database and add the secrets to the namespace as specified [here](https://gitlab.cee.redhat.com/service/app-interface#manage-rds-databases-via-app-interface-openshiftnamespace-1yml).

Reply:
This is just for initial testing in the ephemeral environment; we will remove content from here for the higher environments